src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -94,14 +94,27 @@ export class AddressForm extends React.Component<
 
   render() {
     const lockCountryToOrigin = !this.props.billing && this.props.domesticOnly
+
+    // FIXME: There's an existing iOS bug which uppercases the first two letters when autoCapitalization="word" is enabled
+    // so we are for now disabling autoCapitalization on mobile
+    // https://bugs.webkit.org/show_bug.cgi?id=157410
+    let autoCapitalizeWord = null
```

Comment:
I think setting this to `undefined` here might be more semantically correct (in terms of being the same as not passing the prop in at all).
```suggestion
    let autoCapitalizeWord = undefined
```